app/controllers/npq_separation/admin/admins_controller.rb

```diff
@@ -1,3 +1,61 @@
-class NpqSeparation::Admin::AdminsController < ApplicationController
-  def index = head(:method_not_allowed)
+class NpqSeparation::Admin::AdminsController < NpqSeparation::AdminController
+  before_action :require_super_admin
+
+  def index
+    @admins = Admin.all
+  end
+
+  def new
+    @admin = Admin.new
+  end
+
+  def create
+    @admin = Admin.new(admin_params)
+
+    if @admin.save
+      flash[:success] = t(".success", email: @admin.email)
+      redirect_to action: :index
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    @admin = Admin.find(params[:id])
+
+    if !@admin.super_admin? && @admin.update(super_admin: true)
+      flash[:success] = t(".success", email: @admin.email)
+    else
+      flash[:error] = t(".failure", email: @admin.email)
+    end
+
+    redirect_to action: :index
+  end
+
+  def destroy
+    @admin = Admin.find(params[:id])
+
+    if @admin.super_admin?
```

Comment:
there is logic in the current interface to avoid deleting current admin `current_admin`, should we retain that?